.gitignore

```diff
@@ -28,6 +28,9 @@ vendor/
 .rvmrc*
 .ruby-*
 
+## gemset
```

Comment:
This file is modulesynced from https://github.com/theforeman/foreman-installer-modulesync/blob/master/moduleroot/.gitignore.erb. What produces these gemsets?

Reply:
I did. This is not necessary at all. Will remove it again.